src/main/java/org/gridsuite/study/server/service/PccMinService.java

```diff
@@ -208,4 +211,114 @@ public String getPccMinResource(URI resourcePath) {
         }
         return result;
     }
+
+    public UUID createPccMinParameters(String parameters) {
+        var path = UriComponentsBuilder
+            .fromPath(DELIMITER + PCC_MIN_API_VERSION + "/parameters")
+            .buildAndExpand()
+            .toUriString();
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        HttpEntity<String> httpEntity = new HttpEntity<>(parameters, headers);
+
+        UUID parametersUuid;
+        try {
+            parametersUuid = restTemplate.exchange(pccMinServerBaseUri + path, HttpMethod.POST, httpEntity, UUID.class).getBody();
+        } catch (HttpStatusCodeException e) {
+            throw handleHttpError(e, CREATE_PCC_MIN_PARAMETERS_FAILED);
+        }
+        return parametersUuid;
+    }
+
+    public void updatePccMinParameters(UUID parametersUuid, @Nullable String parameters) {
+        var uriBuilder = UriComponentsBuilder.fromPath(DELIMITER + PCC_MIN_API_VERSION + "/parameters/{uuid}");
+        String path = uriBuilder.buildAndExpand(parametersUuid).toUriString();
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+        HttpEntity<String> httpEntity = new HttpEntity<>(parameters, headers);
+
+        try {
+            restTemplate.put(pccMinServerBaseUri + path, httpEntity);
+        } catch (HttpStatusCodeException e) {
+            throw handleHttpError(e, UPDATE_PCC_MIN_PARAMETERS_FAILED);
+        }
+    }
+
+    public UUID getPccMinParametersUuidOrElseCreateDefaults(StudyEntity studyEntity) {
+        if (studyEntity.getPccMinParametersUuid() == null) {
+            studyEntity.setPccMinParametersUuid(createDefaultPccMinParameters());
+        }
+        return studyEntity.getPccMinParametersUuid();
+    }
+
+    public UUID createDefaultPccMinParameters() {
+        var path = UriComponentsBuilder
+            .fromPath(DELIMITER + PCC_MIN_API_VERSION + "/parameters/default")
+            .buildAndExpand()
+            .toUriString();
+
+        UUID parametersUuid;
+        try {
+            parametersUuid = restTemplate.exchange(pccMinServerBaseUri + path, HttpMethod.POST, null, UUID.class).getBody();
```

Comment:
```suggestion
            return restTemplate.exchange(pccMinServerBaseUri + path, HttpMethod.POST, null, UUID.class).getBody();
```
no need for local var